### DISCUSSION TITLE
2.3.0 Beta 1

### DISCUSSION THREAD

**Discussion by flavioislima** [Heroic-Games-Launcher/HeroicGamesLauncher#1233] [Q&A]
Hello!
Welcome to the first Beta for version 2.2.3 of Heroic this one has a huge amount of changes so bugs are expected.
Please report new bugs for this version by putting `BETA` under the Issue Title. You can also use the discussion linked to the release :D

## What's Changed
* [Windows] Fix "Show log file in folder" once and for all by @CommandMC in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1100
* [Fix] Not able to enable Steam Runtime by @imLinguin in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1108
* [Linux] Integrate Winetricks by @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1090
* [General] UX improvements in header and scrollbar by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1109
* [General] update .gitignore, UX improvements in webview by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1106
* [General] add .editorconfig, align text and icons in sidebar, fix sidebar shadow, sidebar css clean up by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1105
* [General] UX improvements in loading screens by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1112
* [UI] Add more UI effects, fixes w/ user-select, other tweaks. by @8Bitz0 in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1064
* [General] Improvements in language selector when installing game by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1127
* [General] UX improvements in ActionIcons by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1126
* [Tech ]Refactor Legendary/GOGDL handling by @CommandMC in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1092
* [General] Improvements in install dialog by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1129
* [Fix] Flags not displayed when user don't have a font with flags emoji by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1134
* [Tech] Disable eslint during development, added options to editorconfig by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1132
* [Fix] Override exe file for GOG games, launch exe on prefix now sets working directory to the location of exe file by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1133
* [Fix] Gogdl issues don't fail gracefully #1125 by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1138
* [Fix] Cancel/pause downloading of GOG games by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1137
* [Fix] Dialogs hiding behind app window by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1151
* [Webview] Open game store page in webview and refactor by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1149
* Remove MimeType from Shortcut by @JakobDev in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1172
* [Tech] User credentials are no longer stored in plain text by @dawidgarus in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1167
* Add German translation to .desktop by @JakobDev in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1185
* [Linux] Fix Special characters on Wine Prefix Folder by @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1186
* [UI/UX] Sidebar Improvements, consistency fixes and Advanced Settings by @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1199
* [General] Auto-Update In the background  for AppImages, Windows Setup and Mac DMG by @dragonDScript in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/830
* [UI] Add 'report a problem' action in game screen by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1208
* [FIX] Don't reset the filterText when refreshing the library by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1218
* [UI] Fix grayscale/color for games with logo image by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1217
* [Ui] Add an option to disable the controller navigation by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1222
* [Ui] Order games by status and remove header Filter by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1221
* [macOS] Fix install progress, install failling and no space error message by @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1203
* [UI] Improve logs readability by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1227
* [Gog] Setup improvements by @imLinguin in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1174
* Add option to hide games from the library by @arielj in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1228
* Translations update from Hosted Weblate by @weblate in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1195
* [UI] Small fixes by @flavioislima in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1232

## New Contributors
* @dawidgarus made their first contribution in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1109
* @Pl1y made their first contribution in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1178
* @JakobDev made their first contribution in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1172

**Full Changelog**: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/compare/v2.2.6...v2.3.0-beta.1

<hr /><em>This discussion was created from the release <a href='https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/tag/v2.3.0-beta.1'>2.3.0 Beta 1</a>.</em>

**Comment by ghost** [comment]
release to github as pre-release so the auto updater does its job.
